### ISSUE TITLE
false positive from threat-intelligence arch.yourlabs.org

### ISSUE THREAD

**Issue by Hr46ph** [nextdns/metadata#281]
As far as I can tell this is simply an Arch Linux repository mirror.

Domain from the official Arch Mirror list status page:
https://www.archlinux.org/mirrors/yourlabs.org/1062/

**Comment by lyrad87** [comment]
It's also blocked by the notracking blocklist. I've created a ticket there as well, see https://github.com/notracking/hosts-blocklists/issues/380

**Comment by romaincointepas** [comment]
`yourlabs.org` is blocked by https://www.stopforumspam.com/downloads/toxic_domains_whole.txt. Could you talk to them directly so they can fix it?

**Comment by Hr46ph** [comment]
I see no way how. I browsed their site, I read their FAQ, I get pages full of text but I see no easy way to report something. There's a page where I can supposedly remove my own IP address but that's not what I need. I refuse to register an account and post on their forums for something so trivial. I spent maybe 10 minutes so I admit I didn't try very hard but there's a reason.

**With all due respect**, I use your (paid) service, not theirs. As far as I'm concerned, you're blocking a site that is legit, but you duck and point to some other site that I have no business with. I honestly don't think that's a good way of doing business and dealing with customers.

I also think its a bad idea for a paid service to heavily lean on 3rd party lists and not have some way to control your service reliability.

May I suggest you try to be interdependent from 3rd party lists? What I mean is that you use their lists but filter them with your own whitelist system from reports by your own users. Find a way to share your false positive reports back with them.

I think I've done my part: I've reported it to the service I use (that's you my friend ;-) ).
